README.md

```diff
@@ -85,9 +180,9 @@ Its goal is to be as dependency-less as possible in order to be easily compiled
 
   </details>
 
-- **The _wnfs_ Helper Script**
+- **The _wnfs_ Command**
```

Comment:
Maybe we should rename the script in order to avoid confusion with a potential wnfs CLI that actually reads and writes WNFS similar to what the `ipfs` CLI is doing today.

Or we could go for a `Justfile` which is similar to a makefile, but simpler. We're using it in a lot of other places at fission.

And lastly we could go for a nix version, but that's way more involved to set up compared to a justfile.

Reply:
I was considering make but had issues with it in the past. Just looks like an improved make. I'm going to switch to it later.
I agree with renaming it. Maybe `rs-wnfs` because it  specific to the repo?